docs/src/content/docs/ru/guides/site-search.mdx

```diff
@@ -110,13 +112,62 @@ title: Частично проиндексированная страница
 
 #### Конфигурация DocSearch
 
-Плагин Starlight DocSearch также поддерживает настройку компонента DocSearch с помощью следующих дополнительных опций:
+Плагин Starlight DocSearch поддерживает настройку компонента DocSearch с помощью следующих встроенных опций:
 
 - `maxResultsPerGroup`: Ограничивает количество результатов, отображаемых для каждой группы поиска. По умолчанию `5`.
 - `disableUserPersonalization`: Предотвращает сохранение недавних поисков и избранных элементов пользователя в локальном хранилище. По умолчанию `false`.
 - `insights`: Включает плагин Algolia Insights и отправляет события поиска в ваш индекс DocSearch. По умолчанию `false`.
 - `searchParameters`: Объект, настраивающий [параметры поиска Algolia](https://www.algolia.com/doc/api-reference/search-api-parameters/).
 
+##### Дополнительные опции DocSearch
+
+Необходим отдельный файл конфигурации для передачи параметров функции, таких как `transformItems()` или `resultsFooterComponent()`, в компонент DocSearch.
```

Comment:
suggestion: I think it would sound more natural to flip the order of the sentence a little
```suggestion
Для передачи в компонент DocSearch параметров-функций, таких как `transformItems()` или `resultsFooterComponent()`, необходим отдельный файл конфигурации.
```

Reply:
Можно и так, или «функций с параметрами».